make/uaa-images

```diff
@@ -15,4 +15,4 @@ GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
 unset FISSILE_REPOSITORY
 
 make -C "${GIT_ROOT}/src/uaa-fissile-release" \
-    build
+    images
```

Comment:
This change does not work unless there is a corresponding change to let `make releases` also build the UAA releases. Currently the UAA releases are build by `make images` in the SCF release. ☹️